### PR TITLE
Use proper Whereby domain for api

### PR DESCRIPTION
**Pull request by nandito**
* Replace the test environment api endpoints with the production ones
* Use `alpha` tag for the upcoming package version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@whereby.com/browser-sdk",
-  "version": "2.0.0-beta.0",
+  "version": "2.0.0-alpha",
```
Comment (havardholvik) [inline package.json:3]: Add `.0` after `alpha` so that we can iterate on the tag before moving to beta?

**Review by havardholvik — APPROVED** [review]
✅ See comment

File: package.json (hunk 1 above)
Comment (nandito) [reply package.json:3]: We can still iterate with `alpha1` or `alpha.1`. It's not common to use `0` for `alpha` versons
https://semver.org/#spec-item-9

Comment (havardholvik) [reply package.json:3]: Gotcha. I'm fine with that, just saw this `.x` syntax in some guides 👍

**Pull request merged**